EMS-UI/src/app/triage/triage.component.ts

```diff
@@ -273,7 +268,7 @@ export class TriageComponent implements OnInit {
       }
     });
 
-    dialogRef.afterClosed().subscribe(result => {});
+    dialogRef.afterClosed().subscribe(() => {});
```

Comment:
does this line actually serve a purpose?

Reply:
Good point, hadn't noticed that - nope, no obvious purpose. Will delete.